packages/patrol/example/ios/fastlane/Fastfile

```diff
@@ -0,0 +1,30 @@
+default_platform(:ios)
+
+git_authorization = "#{ENV["GIT_USERNAME"]}:#{ENV["GIT_TOKEN"]}"
+
+platform :ios do
+  desc "Signs app on CI"
+  lane :certificates do
+    setup_ci if ENV['CI']
+
+    app_store_connect_api_key(
+      key_id: ENV["APP_STORE_KEY_ID"],
+      issuer_id: ENV["APP_STORE_ISSUER_ID"],
+      key_content: ENV["APP_STORE_KEY"],
+      in_house: false,
+    )
+
+    match(
+      type: "development",
+      git_basic_authorization: Base64.strict_encode64(git_authorization),
+      app_identifier: ["pl.leancode.patrol.*"]
```

Comment:
Is this the "XC wildcard"? Isn't it deprecated?

Reply:
It is a wildcard, I haven't found anything that it will be deprecated

Reply:
OK. I think the guy (from GitHub Issues or Discussion) who managed to get Patrol building apps for physical iOS on CI was saying something about it being deprecated. I can't find the link now.

Reply:
For future reference: I meant [this discussion](https://github.com/leancodepl/patrol/discussions/1300).